script/dump-db

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -euo pipefail
-
 DUMP_FOLDER="/var/lib/openqa/SQL-DUMPS"
+DAYS_TO_KEEP=7
+[ "$1" = "-h" ] || [ "$1" = "--help" ] && echo "Dump the openQA postgresql database to ${DUMP_FOLDER} and clean-up backups older than ${DAYS_TO_KEEP} days." && exit
```

Comment:
Since we're using Bash here:
```suggestion
[[ $1 = '-h' ] || [[ $1 = '--help' ]] && echo "Dump the openQA PostgreSQL database to ${DUMP_FOLDER} and clean-up backups older than ${DAYS_TO_KEEP} days." && exit
```

Reply:
adjusted. I assume you missed to double `]` at the first test? (I used you version and bash complained ;) )